sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -232,10 +234,12 @@ class SQLMetricsSuite extends SparkFunSuite with SharedSQLContext {
       testSparkPlanMetrics(df2, 1, Map(
         2L -> ("TungstenAggregate", Map(
           "number of input rows" -> 6L,
-          "number of output rows" -> 4L)),
+          "number of output rows" -> 4L,
+          "number of bytes used" -> 17305600L)),
         0L -> ("TungstenAggregate", Map(
           "number of input rows" -> 4L,
-          "number of output rows" -> 3L)))
+          "number of output rows" -> 3L,
+          "number of bytes used" -> 25958400L)))
```

Comment:
Note that I just hard-code the right answer in the test, will add comments or use formula to make it clear why this answer is correct later.